compiler/modules/print.llvm/llvm.bal

```diff
@@ -240,14 +241,18 @@ public class FunctionDefn {
     private Value[] paramValues;
     private Linkage linkage = "external";
     private final Context context;
+    private map<string> nameTranslation = {};
```

Comment:
Can't you make this a `string[]` indexed by the `int` you stored to represent the unnamed identifier?

Reply:
If you need to distinguish the case of a basic block label vs a operand, you can use another parallel array for that.

Reply:
Fixed